Dockerfile

```diff
@@ -44,6 +35,7 @@ RUN        npm ci --only=production
 # not trigger additional install as part of the build
 # in the following step.
 COPY       . /calypso/
+RUN        npm ci --only-production
```

Comment:
Typo here: `--only=production`.

The docs on `npm ci` are pretty bad, it's unclear to me whether it actually respects this command which we've kept from `npm install`.